openwisp_firmware_upgrader/tests/test_models.py

```diff
@@ -1,4 +1,4 @@
-import mock
+from unnittest.mock import patch
```

Comment:
`from unnittest import mock` and avoid changing all the other lines because it's not necessary nor brings any value to do so (actually mock.patch is slightly more explicit than just patch).